### DISCUSSION TITLE
ZFS filesystem support for LXC Snapshot capabilities?

### DISCUSSION THREAD

**Discussion by jarjarfinks** [tteck/Proxmox#2950] [Ideas]
Could you add the option to choose the filesystem type for the LXC so as to enable ZFS for snapshots in PVE?

**Comment by tteck** [comment]
I'm afraid this is not how it works.

**Comment by jarjarfinks** [comment]
Really? I'm a noob with this stuff. When I try to do a snapshot of lxcs via
c4pve auto snap it complains about the feature being "unavailable". Can you
point me in the direction of what I should do to be able to take snapshots
using pve for lxcs? I can't seem to figure out how apart from mentions in
forums discussing the lxc guests being zfs

**Reply by tteck** [reply]
Consider starting over with EXT4 as your underlying file system. I typically advise against ZFS for the average user.

**Comment by jarjarfinks** [comment]
I've got a RAID10 with 12 18TB drives formatted as ZFS and don't have a way
to backup everything at the moment. Planning on building a second and third
system for a Ceph cluster. I'm not a complete noob to Linux or
containerization, I just am still learning LXCs and PVE specifically. Tons
of time using docker and Linux in general. Any suggestions for getting LXCs
to be able to get snapshots made from PVE without having to totally
reformat my system? Also, I'm assuming there is a better method of
communication than a GitHub thread if you're open to chatting further.
Thanks in advance!

**Reply by tteck** [reply]
My focus is on developing application LXC scripts, and maintaining said scripts. I do not specialize in, nor do I have the time to provide support for operational issues. Maybe ask your questions in the Proxcord server on Discord.

**Comment by jarjarfinks** [comment]
Can I run your scripts within an lxc that's already created to install any
particular app?

**Reply by tteck** [reply]
No
